android/settings.gradle

```diff
@@ -21,3 +21,5 @@ include ':react-native-webrtc'
 project(':react-native-webrtc').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-webrtc/android')
 include ':react-native-calendar-events'
 project(':react-native-calendar-events').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-calendar-events/android')
+include ':react-native-webview'
```

Comment:
can you please sort it properly? looks like it was sorted ok until calendar-events was added